ArchiSteamFarm/Bot.cs

```diff
@@ -4114,6 +4120,98 @@ internal sealed class Bot : IDisposable {
 			return responses.Count > 0 ? string.Join("", responses) : null;
 		}
 
+		private async Task<string> ResponsePrivacy(ulong steamID, string privacySettingsText) {
+			if ((steamID == 0) || string.IsNullOrEmpty(privacySettingsText)) {
+				ArchiLogger.LogNullError(nameof(steamID) + " || " + nameof(privacySettingsText));
+				return null;
+			}
+
+			if (!IsMaster(steamID)) {
+				return null;
+			}
+
+			if (!IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			if (privacySettingsText.Length != 6) { // There are only 6 privacy settings
+				return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(privacySettingsText)));
+			}
+
+			foreach (char digit in privacySettingsText) {
+				if ((digit < '1') || (digit > '3')) { // They can be set from 1 to 3 
+					return FormatBotResponse(string.Format(Strings.ErrorIsInvalid, nameof(privacySettingsText)));
+				}
+			}
+
+			// Converting digits to enum
+			Steam.PrivacyResponse privacySettings = new Steam.PrivacyResponse {
+				Comments = (Steam.PrivacyResponse.PrivacySettings.EPrivacySetting) byte.Parse(privacySettingsText[5].ToString()),
```

Comment:
This is wrong on so many levels - `byte.Parse` is unacceptable since you're not handling potential exceptions that don't come only from invalid characters, but a lot of other sources. Moreover, explicit cast doesn't handle invalid numbers, even if you can assume underlying type.

Instead, use the same method as used in `ResponseInput` - `Enum.TryParse` + `Enum.IsDefined` right on the digit.